### PR TITLE
Add media queries for contact page

### PR DESCRIPTION
**Pull request by letsmailvjkumar**
Add media queries for mobile screen

### CODE REVIEW

**Comment by adityaraute** [comment]
@letsmailvjkumar can you update your branch? I don't want the same changes to be committed twice. We've already committed the logo changes, right?

**Comment by letsmailvjkumar** [comment]
I have already updated my branch . Is it not reflected.

File: contact.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
         />
         </a>
         <ul>
+          <!-- <li></li>  -->
```
Comment (adityaraute) [inline contact.html:29]: What's this?

**Review by adityaraute — CHANGES_REQUESTED** [review]
1. 
image
The footer colours are messed up again, different from the desktop view.

2. 
image
The text in this div is overflowing. Either truncate or let it wrap around.

3. 
image
I have to horizontally scroll on the screen. Try to restrict this.

**Comment by letsmailvjkumar** [comment]
Ok I will make the changes.

**Comment by letsmailvjkumar** [comment]
p1
p-2
There is contact details  in above part, then again it is same below. Is this ok

File: contact.html (hunk 1 above)
Comment (letsmailvjkumar) [reply contact.html:29]: For smaller screen it the modal to navigate to different pages. 
a1

**Comment by adityaraute** [comment]
No. Let's make it exactly same to the Home and Activities page.

**Comment by agriyakhetarpal** [comment]
@letsmailvjkumar could you push the change you have mentioned in #84?

**Review by adityaraute — APPROVED** [review]
It's not exactly the same, but I think this looks good enough. I'll merge the changes.

**Comment by adityaraute** [comment]
Thanks a lot for your contribution!

**Pull request merged**